static/js/publisher-pages/pages/Releases/components/releasesHeading.js

```diff
@@ -188,7 +188,7 @@ function ReleasesHeading(props) {
 
   return (
     <div>
-      <div className="l-application">
+      <div>
```

Comment:
nitpick: does this 'div' is necessary or can we delete it?

Reply:
Done